crates/ra_ide/src/completion/complete_trait_impl.rs

```diff
@@ -0,0 +1,377 @@
+//! FIXME: write short doc here
```

Comment:
Let's write a short doc here, describing what's exactly this completion variant does :)

Reply:
Taken care of. Let me know if it's not suitable.